app/controllers/v1/todays_project_controller.rb

```diff
@@ -9,5 +9,12 @@ def show
       sorted_projects = Project.all.sort_by { |project| project.achieved_goal_percentage}
       render json: sorted_projects.last
     end
+
+    def quick_info
+      project_numbers = Project.all.count
+      investors_ammount = Investor.all.sum(:ammount)
+      investors_all = Investor.all.count
+      render json: { total_projects_count: project_numbers, total_ammount_invested: investors_ammount, total_investors_count: investors_all } 
```

Comment:
Why not use the same variable name as the key.

Reply:
it hepls me to recognize the key and & value